src/components/PopularCoursesLayout/PopularLayout.tsx

```diff
@@ -15,6 +20,9 @@ const PopularLayout = () => {
           See the courses most commonly chosen by students
           </StyledFrameText>
           <Slider content={<PopularCourses/>} nameLink='All popular courses'/>
+          <PopularMinSlider/>
+          <PopularMobileSlider/>
+          <StyledSliderLink to="#">All popular courses<LongArrowRight /><ArrowRight/></StyledSliderLink>
       </StyledContainer>
     </StyledPopular>
```

Comment:
also you didn't notice, that sections in mobile and tablet don't have a round border anymore, you need to reset border-radius.